warehouse/jobs/types.go

```diff
@@ -91,5 +91,5 @@ const (
 	MaxQueryRetries       int = 3
 	RetryTimeInterval         = 10 * time.Second
 	MaxAttemptsPerJob     int = 3
-	WhAsyncJobTimeOut         = 10 * time.Second
+	WhAsyncJobTimeOut         = 300 * time.Second
```

Comment:
this change is present in diff PR: https://github.com/rudderlabs/rudder-server/pull/2721